Practice4/Program.cs

```diff
@@ -1,15 +1,40 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Practice4
 {
-    class Program
+    public class Program
     {
-        static void Main(string[] args)
+        public const int MAX_ATTEMPTS = 10;
+        public static void Main(string[] args)
         {
+            CheckIfStringIsAPalindromeInLoop();
+            Console.ReadKey();
+        }
+
+        private static void CheckIfInputStringIsAPalindrome(char []array)
+        {
+            int i = 0;
+            while ((array[i] == array[array.Length - i - 1]) && (i < (array.Length - 1) / 2) && (array.Length > 2) && (array.Length % 2 != 0))
+            {
+                i++;
+            }
+            if (i == (((array.Length - 1) / 2)) && (array.Length > 2) && (array.Length % 2 != 0))
```

Comment:
there are some extra brackets

![image](https://user-images.githubusercontent.com/9382934/118272185-e6974980-b4ca-11eb-9f42-806bb3563d36.png)

Reply:
It looks like something is wrong here

![image](https://user-images.githubusercontent.com/9382934/118272276-0e86ad00-b4cb-11eb-91b6-2c37f72bfa9a.png)